contracts/lib/ClaimBuilder.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import "solidity-bytes-utils/contracts/BytesLib.sol";
+import "../lib/GenesisUtils.sol";
```

Comment:
Better be specific in import if you already started doing the same in other files :)

Reply:
fixed it here - https://github.com/iden3/contracts/pull/160
why I missed it in first plase is because we ignoring solhint checks for `lib` folder (.solhintignore rule). Don't know why lib in there.